### PR TITLE
$user parameter is not optional

### PR DESCRIPTION
**Pull request by olvlvl**

### CODE REVIEW

File: src/AppBundle/Entity/Post.php [hunk 1]
```diff
@@ -139,7 +139,7 @@ public function setAuthorEmail($authorEmail)
      */
     public function isAuthor(User $user = null)
     {
-        return $user->getEmail() == $this->getAuthorEmail();
+        return $user && $user->getEmail() === $this->getAuthorEmail();
```
Comment (xabbuh) [inline src/AppBundle/Entity/Post.php]: Should be `null !== $user` according to the Symfony CS.

Comment (xabbuh) [inline src/AppBundle/Entity/Post.php]: Though I think it doesn't make sense to allow `null` here.

Comment (javiereguiluz) [inline src/AppBundle/Entity/Post.php]: I agree. The `null` doesn't make sense.

Comment (olvlvl) [inline src/AppBundle/Entity/Post.php]: Yeah, the ` = null` part should be removed, right?

Comment (javiereguiluz) [inline src/AppBundle/Entity/Post.php]: @olvlvl indeed I think it should be removed. Thanks for updating your branch!

Comment (olvlvl) [inline src/AppBundle/Entity/Post.php]: Here you go :)

**Comment by javiereguiluz** [comment]
Thank you @olvlvl.

**Pull request merged**